com.microsoft.mrtk.spatialmanipulation/Constraints/RotationAxisConstraint.cs.meta

```diff
@@ -0,0 +1,11 @@
+fileFormatVersion: 2
+guid: 4461cef9324cf904299f3d04a3861659
```

Comment:
Just to double check, were these regenerated from MRTK2?

Reply:
Yeah they are regenerated, should they not be?

Reply:
Regeneration is good! Thanks!